src/main/java/Parser.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * A class which stores key words of commands and various comparing methods
+ * used for checking and validation.
+ */
+public class Parser {
+
+    public final static String BYE = "bye";
+    public final static String LIST = "list";
+    public final static String DONE = "done";
+    public final static String DEL = "delete";
+    public final static String DONE_VALID = "done 1";
+    public final static String DEL_VALID = "delete 1";
+    public final static String TODO = "todo";
+    public final static String TODO_VALIDATION = "todo ";
+    public final static String EVT = "event";
+    public final static String EVT_VALIDATION = "event ";
+    public final static String DDL = "deadline";
+    public final static String DDL_VALIDATION = "deadline ";
+    public final static String EMPTY = " ";
+    public final static String BYORAT = "/by ";
```

Comment:
Should underscores be inserted to separate the words in the variable name i.e. "BY_OR_AT"?